pkg/api/summary.go

```diff
@@ -5,29 +5,24 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
-	"time"
 
 	"github.com/wakatime/wakatime-cli/pkg/summary"
 )
 
-const dateFormat = "2006-01-02"
-
-// Summaries fetches summaries for the defined date range.
+// Summary fetches code stats for Today.
```

Comment:
If the endpoint has changed to `/users/current/statusbar/today` also the file might be renamed to `today.go`. What do you think @dron22?

Reply:
Let's leave it `summary.Today` because the response is still a `summary.Summary` struct.